bin/benchpark

```diff
@@ -260,9 +260,7 @@ To build these benchmarks, do the following:
 
     export SPACK_DISABLE_LOCAL_CONFIG=1
 
-    ramble -P -D . workspace setup
-
-Further steps are needed to run the experiments (ramble -P -D . on)
+Further steps are needed to build the experiments (ramble -P -D . workspace setup) and run them (ramble -P -D . on)
```

Comment:
(no change needed) I think I like this organization since everything above this line is something you'd run once (e.g. when you create a new shell) and you don't have to do it again each time you run `ramble workspace setup` and/or `ramble on`.